documentation/migration-guides/Az.13.0.0-migration-guide.md

```diff
@@ -0,0 +1,1784 @@
+    - The parameter Share (alias CloudFileShare) will be deprecated, and ShareClient will be mandatory.
+    - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+
+
+#### Before
+```powershell
+$share = Get-AzStorageShare -Name $shareName -Context $ctx
+$share.CloudFileShare | Get-AzStorageFileHandle -Recursive
+
+$dir = Get-AzStorageFile -ShareName $shareName -Path $path -Context $ctx
+$dir.CloudFileDirectory | Get-AzStorageFileHandle -Recursive
+
+$file = Get-AzStorageFile -ShareName $shareName -Path $filepath -Context $ctx
+$file.CloudFile | Get-AzStorageFileHandle -Recursive
+```
+#### After
+```powershell
+$share = Get-AzStorageShare -Name $shareName -Context $ctx
+$share.ShareClient | Get-AzStorageFileHandle -Recursive
+
+$dir = Get-AzStorageFile -ShareName $shareName -Path $path -Context $ctx
+$dir.ShareDirectoryClient | Get-AzStorageFileHandle -Recursive
+
+$file = Get-AzStorageFile -ShareName $shareName -Path $filepath -Context $ctx
+$file.ShareFileClient | Get-AzStorageFileHandle -Recursive
+```
+
+
+### `Get-AzStorageShare`
+
+- Cmdlet breaking-change will happen to all parameter sets
+  - The child property CloudFileShare from deprecated v11 SDK will be removed. Use child property ShareClient instead.
+  - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+
+
+#### Before
+```powershell
+$share = Get-AzStorageShare -Name $sharename -Context $ctx
+$share.CloudFileShare
+```
+#### After
+```powershell
+$share = Get-AzStorageShare -Name $sharename -Context $ctx
+$share.ShareClient
+```
+
+
+### `New-AzStorageDirectory`
+
+- Cmdlet breaking-change will happen to all parameter sets
+  - The child property CloudFileDirectory from deprecated v11 SDK will be removed. Use child property ShareDirectoryClient instead.
+  - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+
+- Parameter breaking-change will happen to all parameter sets
+  - `-Directory`
+    - The parameter Directory (alias CloudFileDirectory) will be deprecated, and ShareDirectoryClient will be mandatory.
+    - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+  - `-Share`
+    - The parameter Share (alias CloudFileShare) will be deprecated, and ShareClient will be mandatory.
+    - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+
+
+#### Before
+```powershell
+$dir = New-AzStorageDirectory -ShareName testshare1 -Path $dirpath -Context $ctx 
+$dir.CloudFileDirectory
+
+$share = Get-AzStorageShare -Name $sharename -Context $ctx
+New-AzStorageDirectory -Share $share.CloudFileShare -Path $dirpath
+
+$dir = Get-AzStorageFile -ShareName $shareName -Path $path -Context $ctx
+New-AzStorageDirectory -Directory $dir.CloudFileDirectory -Path $dirpath
+```
+#### After
+```powershell
+$dir = New-AzStorageDirectory -ShareName testshare1 -Path $dirpath -Context $ctx 
```

Comment:
```suggestion
$dir = New-AzStorageDirectory -ShareName $sharename -Path $dirpath -Context $ctx 
```

Reply:
@vidai-msft I should've used a variable instead of testshare1 here for -ShareName just to be aligned with other samples... Could you please help fix it? Thanks a lot!